qt/python/mantidqt/mantidqt/dialogs/errorreports/report.py

```diff
@@ -79,6 +81,23 @@ def __init__(self, parent=None, show_continue_terminate=False):
         # Set default focus to the editing box, rather then letting qt try and guess
         self.input_free_text.setFocus()
 
+        self.contact_info_file_path = os.path.join(ConfigService.getAppDataDirectory(),
```

Comment:
Could I suggest that rather than using a custom file and format that we us the `QSettings` functionality? Or possibly I've misunderstood why that couldn't be used?

Reply:
It's much cleaner with QSettings! I've not used them before. I think I've got it working. Do the self.CONTACT,NAME,EMAIL map to their save location?